test.js

```diff
@@ -158,6 +154,27 @@ describe('capture-exit', function() {
         exit.onExit(function () { });
       }).to.throw('Cannot install handler when exit is not captured.  Call `captureExit()` first');
     });
+
+    it('throws if an exit is already happening', function() {
```

Comment:
Ya, I still think throwing (as we do now) is the right answer until we have good use cases as a guide.  I was only suggesting that we _could_ possibly make the second script work (not that we _should_). 😝